paasta_itests/steps/setup_steps.py

```diff
@@ -59,6 +59,23 @@ def setup_system_paasta_config():
                 'user': None,
                 'password': None,
             },
+            'marathon_servers': [
+                {
+                    'url': _get_marathon_connection_string('marathon'),
```

Comment:
I guess I should probably update #1551 to allow a single string in `url`, rather than just a `List[str]`, since MarathonClient allows either, and we generally just pass `url` through unmodified.